utilities/score/README.md

```diff
@@ -0,0 +1,29 @@
+# Score Utility
+
+## Direct run
+
+1\. Install dependencies:
+
+```bash
+virtualenv utilities/score/env
+source utilities/score/env/bin/activate
+pip install -r utilities/score/requirements.txt
+```
+
+2\. Define local environment:
+
+```bash
+export MODELS_DIR=/tmp/models
+export WORKSPACE_DIR=/tmp/workspace
+export CORPUS_DIR=$PWD/test/corpus
```

Comment:
Remove `MODELS_DIR` and `CORPUS_DIR` that are not relevant to this utility.

Reply:
done